### ISSUE TITLE
Chromefy error

### ISSUE THREAD

**Issue by arit98** [imperador/chromefy#152]
i installed chrome os on my amd ryzen with rammus recovery, i tried other recovery also like grunt but they might be successfully not installed yet
right now the thing is chrome os stuck at its official chrome logo
in this case i mentioned that my keyboard is worked that means no hanged but it stucked

![IMG_20200906_151643](https://user-images.githubusercontent.com/58144979/92333024-b7108180-f09f-11ea-910c-6687db47647b.jpg)

**Comment by xeu100** [comment]
How did you even get Ryzen to boot? Zen architecture isn't supported at all by chromeos

**Comment by eugene373** [comment]
It's failing to boot because of video drivers.

**Comment by xeu100** [comment]
> It's failing to boot because of video drivers.

Yes, but previously Ryzen wouldn't even boot with Chromefy or Brunch.

**Comment by eugene373** [comment]
> > It's failing to boot because of video drivers.
> 
> Yes, but previously Ryzen wouldn't even boot with Chromefy or Brunch.

I understand, they have push the source upstream for AMD. Hopefully when the newer AMD chromebooks are released it will bring the video drivers with it.

**Comment by EJ-dot** [comment]
Literally 5 years here its 2025 bro-
